src/clients/security_auto.rb

```diff
@@ -130,10 +120,6 @@ def main
       Builtins.y2debug("ret=%1", @ret)
       Builtins.y2milestone("Security auto finished")
       Builtins.y2milestone("----------------------------------------")
-
-      # is it needed?
-      deep_copy(@ret)
-      # EOF
```

Comment:
this is in fact wrong as you kill return value. I just said that deep_copy is not needed, so still you need to have here `@ret` otherwise it return nil, causing bugs.